salt/modules/junos.py

```diff
@@ -1278,6 +1261,11 @@ def install_config(path=None, **kwargs):
             log.error(message)
             ret["message"] = message
             ret["out"] = False
+        except Exception as exception:  # pylint: disable=broad-except
```

Comment:
```suggestion
        except Exception as exc:  # pylint: disable=broad-except
```
No need to be so verbose and it's also more consistent with the bad majority of our code.
Yes, this module has other bad examples :)